compute/instances.go

```diff
@@ -765,7 +765,7 @@ func (c *InstancesClient) unqualifyNetworking(info map[string]NetworkingInfo) (m
 		if v.Vnic != "" {
 			unq.Vnic = c.getUnqualifiedName(v.Vnic)
 		}
-		if v.Nat != nil {
+		if v.Nat != nil && len(v.Nat) > 0 && v.Nat[0] != "null" {
```

Comment:
The `nil` check is extraneous if also checking `len`

```suggestion
		if len(v.Nat) > 0 && v.Nat[0] != "null" {
```